internal/cypress/config.go

```diff
@@ -26,6 +26,7 @@ type Project struct {
 	Npm            config.Npm         `yaml:"npm,omitempty" json:"npm"`
 	RootDir        string             `yaml:"rootDir,omitempty" json:"rootDir"`
 	RunnerVersion  string             `yaml:"runnerVersion,omitempty" json:"runnerVersion"`
+	DryRun         bool
```

Comment:
This will cause `DryRun` to be serialized/deserialized ⚠️ 
I'm also not convinced that params that aren't part actually part of the config should end up here 🤔

Reply:
updated.